### PR TITLE
autocapitalization added to text fields on sign up vc

### LINKED ISSUE #59: As a user, I want the name and orgnanization fields to capitalise the first letter of my entry, so that my sign up process is more streamlined
_As a user, I want the name and orgnanization fields to capitalise the first letter of my entry, so that my sign up process is more streamlined_

**Acceptance Criteria:**
When I enter my first name on the sign up screen, is the first letter only auto-capitalised
When I enter my last name on the sign up screen ,is the first letter only auto-capitalised
When I enter my organisation on the sign up screen, is the first letter only auto-capitalised

**Solution Guidence:** 
Setting the correct `textContentType` on the fields should handle this for names. 
Otherwise use the `autocapitalizationType`.

### PR DESCRIPTION
**Pull request by ClarissaVinciguerra**

### CODE REVIEW

File: TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ class SignUpViewController: UIViewController, KeyboardDismissing, AlertPresentin
             firstNameTextField.delegate = self
             firstNameTextField.keyboardType = .default
             firstNameTextField.returnKeyType = .next
+            firstNameTextField.autocapitalizationType = .words
```
Comment (AlexCornforth) [inline TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift:36]: Did you try setting the `textContentType` to `.name` for this?

**Review by AlexCornforth — COMMENTED** [review]
Looks good - just quick question about trying `textContentType`.

File: TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift (hunk 1 above)
Comment (ClarissaVinciguerra) [reply TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift:36]: It seems that setting the textContentType doesn't autocapitalize on its own - unless on a device with a keyboard it provides a capital letter first. Would you like me to make the changes with both .autocapitalizationType and .textContentType?

Comment (AlexCornforth) [reply TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift:36]: Ah ok cool - perfect as it is then!

**Review by AlexCornforth — APPROVED** [review]
LGTM 👍

**Pull request merged**